### ISSUE TITLE
Notifications are being shown as warnings

### ISSUE THREAD

**Issue by AwaNoodle** [lholman/OneBuild#24]
Notifications about actions inside of the tasks are being reported back as warnings rather than info. For example:

**Invoke-Compile**
```powershell
WARNING: Using Configuration mode 'Debug'. Modify this by passing in a value for the parameter '-configMode'
```

**Invoke-UnitTests**
```powershell
WARNING: Adding: C:\OneBuildTest\OneBuildTest.MoreNunitTests\bin\OneBuildTest.MoreNunitTests.dll
WARNING: Adding: C:\OneBuildTest\OneBuildTest.NunitTests\bin\OneBuildTest.NunitTests.dll
WARNING: Removing all previous test result file(s) matching: C:\OneBuildTest\TestResult*.xml
```

**Comment by lholman** [comment]
Well spotted @AwaNoodle 
Seeing as all build logic is encapsulated within advanced Powershell Modules I'm thinking using the standard -verbose parameter along with Write-Verbose would make sense for the majority of verbose/debug style output

**Comment by AwaNoodle** [comment]
Makes sense to me too :)

**Comment by lholman** [comment]
@AwaNoodle This is fixed with PR #34 and a minor merge conflict commit.
